src/test/java/org/mule/tools/artifact/archiver/internal/PackageBuilderTest.java

```diff
@@ -197,94 +206,94 @@ public void createDeployableFileSettingPluginsFolderTest() throws IOException {
-    Assert.assertEquals("Warning about empty directory was not logged", EXPECTED_EMPTY_DIRECTORY_MESSAGE,
-                        logCaptor.getValue().getRenderedMessage());
-  }
-
-  @Ignore
-  @Test
-  public void createArtifactWrongDirectoryStructureLoggingTest() throws IOException {
-    Logger.getRootLogger().addAppender(appender);
-
-    File emptyDirectoryMock = mock(File.class);
-    when(emptyDirectoryMock.isDirectory()).thenReturn(true);
-    when(emptyDirectoryMock.exists()).thenReturn(true);
-
-    File childFileMock = mock(File.class);
-
-    File[] wrongDirectoryStructure = new File[1];
-    wrongDirectoryStructure[0] = childFileMock;
-
-    when(emptyDirectoryMock.listFiles()).thenReturn(wrongDirectoryStructure);
-
-    when(destinationFileMock.exists()).thenReturn(false);
-
-    this.packageBuilder.generateArtifact(emptyDirectoryMock, destinationFileMock);
-
-    verify(appender).doAppend(logCaptor.capture());
-    Assert.assertEquals("Warning about wrong directory structure was not logged", EXPECTED_WRONG_DIRECTORY_STRUCTURE_MESSAGE,
-                        logCaptor.getValue().getRenderedMessage());
-  }
-
-  @Ignore
-  @Test
-  public void generateArtifactGivenCorrectStructureTest() throws IOException {
-    Logger.getRootLogger().addAppender(appender);
-
-    File targetFolder = mock(File.class);
-
-    File[] filesInTargetDirectoryMock = getFiles();
-    when(targetFolder.exists()).thenReturn(true);
-    when(targetFolder.isDirectory()).thenReturn(true);
-    when(targetFolder.listFiles()).thenReturn(filesInTargetDirectoryMock);
-
-    when(destinationFileMock.exists()).thenReturn(false);
-    when(destinationFileMock.getName()).thenReturn("app.zip");
-
-    MuleArchiver archiverMock = mock(MuleArchiver.class);
-    this.packageBuilder.withArchiver(archiverMock);
-
-    this.packageBuilder.generateArtifact(targetFolder, destinationFileMock);
-
-    verify(appender).doAppend(logCaptor.capture());
-    String SUCCESS_MESSAGE = "File " + destinationFileMock.getName() + " has been successfully created";
-    Assert.assertEquals("Info about created destination file not logged", SUCCESS_MESSAGE,
-                        logCaptor.getValue().getRenderedMessage());
-  }
+  // @Test(expected = IllegalArgumentException.class)
+  // public void createArtifactNullDirectory() throws IOException {
+  // File nullDirectory = null;
+  // this.packageBuilder.generateArtifact(nullDirectory, destinationFileMock);
+  // }
+  //
+  // @Test(expected = IllegalArgumentException.class)
+  // public void createArtifactNotADirectory() throws IOException {
+  // File notADirectory = mock(File.class);
+  //
+  // this.packageBuilder.generateArtifact(notADirectory, destinationFileMock);
+  // }
+  //
+  // @Test(expected = IllegalArgumentException.class)
+  // public void createArtifactInexistentDirectory() throws IOException {
+  // File inexistentDirectory = mock(File.class);
+  // when(inexistentDirectory.exists()).thenReturn(false);
+  //
```

Comment:
done, I ended up removing them as they used an already deleted method and they validated logic of and old phase